views/view_product_filter.xml

```diff
@@ -0,0 +1,17 @@
+<odoo>
+    <data>
+        <record model='ir.ui.view' id='view_product_filter'>
+            <field name='name'>product.template.search.inherit</field>
+            <field name='model'>product.template</field>
+            <field name='inherit_id' ref='product.product_template_search_view'/>
+            <field name='arch' type='xml'>
+                <field name="name" position="after">
+                    <filter string="Subscriptions" name="sub" domain="[('type','=','sub')]"/>
```

Comment:
Bien! Después checa este link mencionan que se pueden hacer con la interfaz también los filtros
https://www.odoo.com/forum/help-1/question/how-to-add-custom-filter-25234

Reply:
si vi la opción, creo que es buena pero si debe haber algunos predeterminados como suscripción